test/e2e/v1beta1/scripts/gh-actions/run-e2e-tune-api.py

```diff
@@ -78,11 +139,19 @@ def objective(parameters):
         namespace_labels['katib.kubeflow.org/metrics-collector-injection'] = 'enabled'
         client.CoreV1Api().patch_namespace(args.namespace, {'metadata': {'labels': namespace_labels}})
 
-    # Test with run_e2e_experiment_create_by_tune
-    exp_name = "tune-example"
+    # Test with run_e2e_experiment_create_by_tune_default_metrics_collector 
+    # and run_e2e_experiment_create_by_tune_custom_metrics_collector
     exp_namespace = args.namespace
     try:
-        run_e2e_experiment_create_by_tune(katib_client, exp_name, exp_namespace)
+        exp_name = "tune-example-default-metrics-collector"
+        logging.info(f"Runnning E2E for Experiment created by tune: {exp_namespace}/{exp_name}")
+        run_e2e_experiment_create_by_tune_default_metrics_collector(katib_client, exp_name, exp_namespace)
+        logging.info("---------------------------------------------------------------")
+        logging.info(f"E2E is succeeded for Experiment created by tune: {exp_namespace}/{exp_name}")
+
+        exp_name = "tune-example-custom-metrics-collector"
+        logging.info(f"Runnning E2E for Experiment created by tune: {exp_namespace}/{exp_name}")
+        run_e2e_experiment_create_by_tune_custom_metrics_collector(katib_client, exp_name, exp_namespace)
```

Comment:
Maybe splitting these two e2e tests into separate `try-catch` clauses is better to identify error :)